src/Controls/src/Core/HandlerImpl/ContentPage.Impl.cs

```diff
@@ -49,6 +49,8 @@ protected override void InvalidateMeasureOverride()
 			}
 		}
 
+		public override int GetContentTypeHashCode() => TypeHashCode ?? (this.GetType(), Content.GetContentTypeHashCode()).GetHashCode();
```

Comment:
Should this be a `??=`? Are you wanting to set/cache the value from `GetHashCode()`